microsoftgraph/microsoft-graph-explorer
# Fixes the focus order of the share query/link button when navigating with  keyboard.

## Overview

Fixes the focus order of the share query/link button when navigating with  keyboard.

### Demo

N/A

### Notes

N/A

## Testing Instructions

1. Open Graph Explorer
3. Navigate to the 'Share Query' button through keyboard.
4. Observe the focus order.
